docs/book/installation/requirements.rst

```diff
@@ -69,6 +69,16 @@ By default, the database connection is pre-configured to work with a following M
 
     You might also use any other RDBMS (like PostgreSQL), but our database migrations support MySQL only.
 
+NPM Package Manager
+-------------------
+
+Sylius Frontend depends on `npm packages <https://docs.npmjs.com/about-npm>`_ for it to run you need to have Node.js installed.
+Current minimum supported version of Node.js is:
+
++---------------+-----------------------+
+| Node.js       | 14.0+                 |
```

Comment:
As version `16.x` doesn't work (from what I remember) writing the required version as `14.0+` may suggest everything above 14 is working (v16 including), so I suggest writing it as below:
```suggestion
| Node.js       | 14.x                 |
```